flink-contrib/docker-flink/docker-compose.yml

```diff
@@ -20,16 +20,20 @@ version: "2"
 services:
   jobmanager:
     image: flink
+    container_name: "jobmanager"
+    expose:
+      - "6123"
     ports:
       - "48081:8081"
     command: jobmanager
-    volumes:
-      - /opt/flink/conf
 
   taskmanager:
     image: flink
+    expose:
+      - "6121"
+      - "6122"
```

Comment:
Sure, makes sense since those ports are not reachable by TaskManagers running in different containers.